index.bs

```diff
@@ -698,6 +701,7 @@ The <dfn method for=CookieStore>set(|options|)</dfn> method steps are:
 1. Let |origin| be |settings|'s [=environment settings object/origin=].
 1. If |origin| is an [=opaque origin=], then return [=a promise rejected with=] a "{{SecurityError}}" {{DOMException}}.
 1. Let |url| be |settings|'s [=environment/creation URL=].
+1. Let |partitioned| be true if |options|["{{CookieInit/partitioned}}"] is present and true, or false otherwise.
```

Comment:
Since this defaults to true now, this step can be removed...

Reply:
Done.